services/director-v2/src/simcore_service_director_v2/models/schemas/clusters.py

```diff
@@ -39,7 +39,7 @@ class WorkerMetrics(BaseModel):
     task_counts: TaskCounts = Field(..., description="task details")
 
 
-AvailableResources = DictModel[str, PositiveFloat]
+AvailableResources: TypeAlias = DictModel[str, PositiveFloat]
```

Comment:
THOUGHT: I thought we wanted to get rid of these ``DictModel``s  by using directly ``dict[str, PositiveFloat]``.

Reply:
If we want to do that, I guess adding a case will remind us to deprecate it